cpu/esp32/vendor/esp-idf/esp_funcs.c

```diff
@@ -83,7 +84,7 @@ void _esp_error_check_failed(esp_err_t rc, const char *file, int line,
  */
 uint32_t IRAM_ATTR esp_log_timestamp(void)
 {
-    return system_get_time() / USEC_PER_MSEC;
+    return system_get_time() / US_PER_MS;
```

Comment:
Not sure whether it makes sense. It's vendor code. Each change makes a later upgrade to newer version more difficult.

Reply:
Ok, make sense reverted